examples/dt/uni02beta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinder:
+    extraMounts:
+      - extraVol:
+          - propagation:
+              - CinderVolume
+            extraVolType: Undefined
+            volumes:
+              - name: nfs-shares
+                projected:
+                  sources:
+                    - secret:
+                        name: cinder-volume-ontap-shares-secrets
+            mounts:
+              - name: nfs-shares
+                mountPath: /etc/cinder/nfs_shares.d
+                readOnly: true
+
+  cinderVolumes:
+    ontap-nfs:
+      customServiceConfig: |
+        [ontap]
+        volume_backend_name=ontap
+        volume_driver=cinder.volume.drivers.netapp.common.NetAppDriver
+        netapp_server_hostname=_replaced_  # CHANGEME
+        netapp_server_port=80
+        netapp_storage_protocol=nfs
+        netapp_storage_family=ontap_cluster
+        nfs_shares_config=/etc/cinder/nfs_shares.d/shares-config
+    networkAttachments:
+      - storage
+    customServiceConfigSecrets:
+      - cinder-volume-ontap-secrets
```

Comment:
The values are correct, but the lines need to be indented under the optap-nfs section.

Reply:
Thank you for the good eye. Fixed.